brainiak/utils/fmrisim.py

```diff
@@ -1444,42 +1789,56 @@ def _generate_noise_temporal(stimfunction_tr,
     # Finally, z score each voxel so things mix nicely
     noise_temporal = stats.zscore(noise_temporal, 3)
```

Comment:
zscoring when some variables have 0 variance may cause some warnings to be thrown - not necessarily a problem, but could be solved by using a masked array

Reply:
Yes good point. I will put this on the list for future things to fix